### ISSUE TITLE
Apps wrongly deployed after host reboot

### ISSUE THREAD

**Issue by shrmnk** [dokku/dokku#2999]
### Description of problem
After the host reboots, the apps are wrongly deployed. 

I have 2 apps currently on the site:
The first a Rails+Postgres app `dashboard`, with the main domain added to it. (e.g. example.com)
The second a [static site](https://github.com/shrmnk/dokku-static-site) `mqtt-web`, with a subdomain added to it. (e.g. mqtt.example.com)

After rebooting, it appears that the subdomain now points the first app, thus generating a Rails "We're sorry, but something went wrong." page when visited.
The main domain doesn't work with a Bad Gateway error.

To fix this, I have to do a `dokku ps:restartall`. 

This is reproducible every time I restart the server, and will require me to do the above fix before the apps are properly deployed to the respective domains.

---

[Output of `dokku report` and `docker ps -a` before/after the `dokku ps:restartall`](https://gist.github.com/shrmnk/a3a8f2b2adf4f977982bd4a267c240ae)

**Environment details (AWS, VirtualBox, physical, etc.):** AWS EC2 t2.micro, Ubuntu 16.04

**How was Dokku installed?:** As per README.md
```
wget https://raw.githubusercontent.com/dokku/dokku/v0.10.5/bootstrap.sh
sudo DOKKU_TAG=v0.10.5 bash bootstrap.sh
```

**How reproducible:** Always. Happens every `sudo reboot now`.

**Steps to Reproduce:**
1. `sudo reboot now`
2. Visit domain & subdomain

**Comment by josegonzalez** [comment]
Closed by #2736, which will be packaged in the next minor release.

**Comment by benwinding** [comment]
For anyone in my situation. My server was restarted unexpectedly (Digital Ocean), and some of my Dokku apps were redirecting to the wrong sub-domains. This command fixed all of them though:

`dokku ps:rebuildall`

Running Dokku Version 0.10.5

**Comment by jalcine** [comment]
Sounds like something one can place into a cronjob for system starts.

Nice catch!

**Comment by josegonzalez** [comment]
@benwinding They would redirect for a short amount of time. The `ps:restore` command actually rebuilds the routing for a given container and then rebuilds any missing applications, but it _does_ take time. This behavior was implemented in `0.11.0` - we're on `0.11.6` and will be shortly making another release - so please keep your Dokku installation up to date as it would have fixed this issue.

We previously used `ps:rebuildall` on the restore command - it happens at boot - but it was changed to restore [here](https://github.com/dokku/dokku/pull/1613). `ps:restore` is much faster and does the right thing, but it's not instantaneous, so there will be a period of time where the server does not serve the right thing.

For anyone following along - Upgrade your installations.
